src/osemgrep/cli_login/Console_Spinner.ml

```diff
@@ -0,0 +1,28 @@
+(*****************************************************************************)
+(* Console Experience *)
+(*****************************************************************************)
+
+let spinner = [| "⠋"; "⠙"; "⠹"; "⠸"; "⠼"; "⠴"; "⠦"; "⠧"; "⠇"; "⠏" |]
+
+(*
```

Comment:
it's usually better to add comments that are useful for the user of the API in the .mli, and comments that are useful for the implementer of the API in the .ml

OCaml programmers tend to rely a lot on the .mli to understand how to use a module; they don't want to look at the implementation, they just want to look at the .mli that should contain enough with the types and comments to explain how to use the module.

Reply:
ack